### DISCUSSION TITLE
Am I right that I cant use this library for a closed source Xamarin.Forms (iOS, Android, UWP) app?

### DISCUSSION THREAD

**Discussion by Agredo** [Mapsui/Mapsui#1234] [Q&A] [answered]
Currently I am struggeling to understand how to use your library in a closed source Xamarin.Forms app. 

It would be necessary to make it possible for the user to replace MapsUI binaries but I dont see a way for iOS published in app store. 

I think I cant use this library but maybe someone already worked on a closed source Xamarin.Forms app and can explain me how  I have to handle this. 

Nevertheless: Thank you for this great librarie! For coming OS projects where I need maps I will definetley use this library!

Thanks
Agredo

**Comment by pauldendulk** [comment, marked as answer]
We are planning to change the license to MIT. See discussion here:

https://github.com/Mapsui/Mapsui/issues/301

Almost all original contributors have signed the CLA to allow such a change. One thing that needs to be removed is Mapsui.Geometries which will be replaced with NTS. 

We can not say anything about when this will be done however.

**Reply by Agredo** [reply]
Thank you :) Sounds promising :)
